examples/getstarted/config/features.js

```diff
@@ -0,0 +1,5 @@
+module.exports = ({ env }) => ({
+  future: {
+    unstable_contentReleases: env.bool('FUTURE_UNSTABLE_CONTENT_RELEASES', false),
```

Comment:
Hmmm the var name is a tad long do we have other alternatives such as those defined in the env var RFC?

Reply:
IMO, I don't think there's a problem with long variable names - it's a definite personal thing 😅

Reply:
Considering these aren't supposed to be friendly, because they're very use-at-your-own-risk, I'm fine with it being long

However, the env var thing Derrick is referencing brings up a good point. Can we avoid using _ here so that we can continue using it as a delimiter? Either `unstable: { contentReleases }` or `unstableContentReleases` would work.

Or, my preference is that "future" is stable features we can't release due to breaking changes, and "unstable" is a totally separate grouping for unstable/development features.

And also relating to that RFC, could you use a format like STRAPI_FUTURE_UNSTABLECONTENTRELEASES? STRAPI_ prefix, underscore means child property? That's the env value that would be loaded into that property automatically even if that line were removed from the config so it would be nice to have it identical for an example.

Reply:
> However, the env var thing Derrick is referencing brings up a good point. Can we avoid using _ here so that we can continue using it as a delimiter? Either unstable: { contentReleases } or unstableContentReleases would work.

Ah i misunderstood Derrick in that case, yeah i think `unstableContentReleases` works. 

> Or, my preference is that "future" is stable features we can't make default due to breaking changes, and "unstable" is a totally separate grouping for unstable/development features.

Hm, i think that then creates a weird area where we have stable features we're not releasing for XYZ... IMO it should just be unstable even if we're not making changes and if we're not releasing them because of breaking changes then they become `vXContentReleases`

> And also relating to that RFC, could you use a format like STRAPI_FUTURE_UNSTABLECONTENTRELEASES? STRAPI_ prefix, underscore means child property? That's the env value that would be loaded into that property automatically even if that line were removed from the config so it would be nice to have it identical for an example.

I don't fully understand why we would want to do that (maybe more explanation would be helpful?) but looking at other ENVs it matches them e.g. `TRANSFER_TOKEN_SALT` or `API_TOKEN_SALT` no?

Reply:
It doesn't really matter, it's just a visual/example thing. But we will be mapping env vars to config values automatically, case-insensitive, with _ denoting a child object, so keeping with that pattern for the ones being manually loaded would be nice.

Reply:
I'm happy to go with the flow, so lets do your suggestion.